akvo/rsr/filters.py

```diff
@@ -98,26 +98,45 @@ def get_locations(location, locations):
 def location_choices(qs):
     """From a queryset get possible location filter choices"""
 
-    if qs.model is Project:
-        location_model = ProjectLocation
-    elif qs.model is ProjectUpdate:
+    country_ids = (
+        get_recipient_country_ids(qs) if qs.model is Project else
```

Comment:
If we change the list of locations shown in the project listing in the directory, should we also change the list of locations in the project page's header? 

![image](https://cloud.githubusercontent.com/assets/315678/24896499/d850b5ee-1eb2-11e7-8080-88c08b951026.png)